src/main/java/com/couchbase/lite/replicator/ChangeTracker.java

```diff
@@ -326,7 +326,8 @@ public void process(HttpRequest request, HttpContext context) throws HttpExcepti
                 Log.v(Log.TAG_CHANGE_TRACKER, "%s: Making request to %s", this, maskedRemoteWithoutCredentials);
                 HttpResponse response = httpClient.execute(request);
                 StatusLine status = response.getStatusLine();
-                if (status.getStatusCode() >= 300 && !Utils.isTransientError(status)) {
+                if ((status.getStatusCode() >= 300 && !Utils.isTransientError(status)) ||
+                        status.getStatusCode() >= 300 && mode != ChangeTrackerMode.LongPoll) {
```

Comment:
@pasin, sorry, you are right. this check was not correct. Thank you very much for catch my coding mistake.